tests/js/token.scrypttest.js

```diff
@@ -1,9 +1,9 @@
 const { expect } = require('chai');
 const { bsv, buildContractClass, signTx, toHex, getPreimage, num2bin, Sig, PubKey, SigHashPreimage } = require('scryptlib');
-const { inputIndex, inputSatoshis, tx, compileContract, DataLen } = require('../../helper');
+const { inputIndex, inputSatoshis, newTx, compileContract, DataLen } = require('../../helper');
 
 // make a copy since it will be mutated
```

Comment:
Delete. Same for every scrypttest.

Reply:
9a531461eca019f0a14ad12158cfe4e098d0e462